ark/api/v1alpha1/query_types.go

```diff
@@ -119,3 +114,9 @@ type QueryList struct {
 func init() {
 	SchemeBuilder.Register(&Query{}, &QueryList{})
 }
+
+type Response struct {
+    Target      QueryTarget `json:"target,omitempty"`
+    Content     string      `json:"content,omitempty"`
```

Comment:
This is great; can we put a comment (ideally in the crd docs as well, i believe there is annotation or similar we have here) that this is deprecated and will likely be replaced in the future